src/lib.rs

```diff
@@ -326,6 +356,73 @@ impl PrometheusMetricsBuilder {
     }
 }
 
+#[derive(Debug)]
+/// Configuration for a single metric
+///
+/// Allows configuring name and labels set for the metric
+pub struct ActixMetric {
+    name: String,
+    labels: Vec<String>,
+}
+
+impl ActixMetric {
+    /// Create a new metric configuration
+    pub fn new(name: &str, labels: Vec<&str>) -> ActixMetric {
```

Comment:
Could you add an implementation of [`From`](https://www.reddit.com/r/rust/comments/f55ur9/fromstr_vs_fromstr_vs_fromstring_which_should_i/?onetap_auto=true) to just add a name?

Reply:
Would you then default the labels to something or leave them empty? I'm not 100% sure `From` is the best here as in my opinion it's not clear that the `From` implementation would do with labels. Maybe some ugly name like `new_with_default_labels`  would work & have the clarity.